drivers/hud_lsa/app/models/hud_lsa/generators/fy2023/missing_data_concern.rb

```diff
@@ -143,6 +146,17 @@ def missing_data(user)
     )
   end
 
+  private def missing_inventory_household_types(user)
+    missing_data_rows(
+      GrdaWarehouse::Hud::Inventory.joins(project: :organization).
+      includes(project: :funders).
```

Comment:
For clarity, maybe better to use join or preload explicitly rather than `includes()`?

Reply:
Hmm, I can't remember the reasoning behind not just throwing the `:funders` into the joins above.  Maybe left-outer joins?  It doesn't look like we use it explicitly, but I think we're using it to limit something. That said, it could totally be a copy/paste issue.  I'll investigate further, thanks for noticing.